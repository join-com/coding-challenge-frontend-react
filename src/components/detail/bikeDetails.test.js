import React from 'react';
import { shallow, mount } from 'enzyme';
import BikeDetail from './bikeDetail';
import { Helmet } from "react-helmet";
import { Provider } from "react-redux";
import configureMockStore from "redux-mock-store";

const mockStore = configureMockStore();
const store = mockStore({
    bikeList: {
        stolenBikeDetail: {},
        stolenBikeMap: {}
    }
});
const match = {
    params: {
        id: 23
    }
}

describe('Bike Details', () => {
    it('Bike Details should render correctly', () => {
        const component = mount(
            <Provider store={store}>
                <BikeDetail match={match} />
            </Provider>
        );
        expect(component).toMatchSnapshot();
    });
});