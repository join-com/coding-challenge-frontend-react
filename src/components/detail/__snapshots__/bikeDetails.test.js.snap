// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bike Details Bike Details should render correctly 1`] = `
<Connect(BikeDetail)
  classes={
    Object {
      "card": "Connect-BikeDetail--card-4",
      "content": "Connect-BikeDetail--content-6",
      "cover": "Connect-BikeDetail--cover-7",
      "descriptions": "Connect-BikeDetail--descriptions-10",
      "detailScreen": "Connect-BikeDetail--detailScreen-8",
      "details": "Connect-BikeDetail--details-5",
      "error": "Connect-BikeDetail--error-12",
      "fullPageHeight": "Connect-BikeDetail--fullPageHeight-2",
      "mainContainer": "Connect-BikeDetail--mainContainer-1",
      "map": "Connect-BikeDetail--map-9",
      "subHeading": "Connect-BikeDetail--subHeading-11",
      "totalCasesText": "Connect-BikeDetail--totalCasesText-3",
    }
  }
/>
`;
